### PR TITLE
Add projects list

### PR DESCRIPTION
**Pull request by cygaar**
To make it easier to add your project to the list of projects using ERC721A, we've put together a markdown file of all the projects that have let us know about their implementation. Anyone is allowed to add their project to the list!

### CODE REVIEW

File: projects.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## ERC721A Implementations
+
+Here are a list of projects that have or will be implementing ERC721A as part of their mint process. If your project is using ERC721A, feel free to open a PR and add it to the list!
+
+- Azuki | [Etherscan](https://etherscan.io/address/0xed5af388653567af2f388e6224dc7c4b3241c544) | [Twitter](https://twitter.com/AzukiZen) 
+- Cereal Club | Etherscan | [Twitter](https://twitter.com/cerealclubnft)
```
Comment (NeoDigi) [inline projects.md:6]:
```suggestion
- [Cereal Club](https://www.cerealclub.io/) | Etherscan | [Twitter](https://twitter.com/cerealclubnft)
```

File: projects.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+## ERC721A Implementations
+
+Here are a list of projects that have or will be implementing ERC721A as part of their mint process. If your project is using ERC721A, feel free to open a PR and add it to the list!
+
+- Azuki | [Etherscan](https://etherscan.io/address/0xed5af388653567af2f388e6224dc7c4b3241c544) | [Twitter](https://twitter.com/AzukiZen)
```
Comment (NeoDigi) [inline projects.md:5]:
```suggestion
- [Azuki](https://www.azuki.com/) | [Etherscan](https://etherscan.io/address/0xed5af388653567af2f388e6224dc7c4b3241c544) | [Twitter](https://twitter.com/AzukiZen) 
```

File: projects.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+## ERC721A Implementations
+
+Here are a list of projects that have or will be implementing ERC721A as part of their mint process. If your project is using ERC721A, feel free to open a PR and add it to the list!
+
+- Azuki | [Etherscan](https://etherscan.io/address/0xed5af388653567af2f388e6224dc7c4b3241c544) | [Twitter](https://twitter.com/AzukiZen) 
+- Cereal Club | Etherscan | [Twitter](https://twitter.com/cerealclubnft)
+- The Lost Glitches | Etherscan | [Twitter](https://twitter.com/TheLostGlitches)
```
Comment (NeoDigi) [inline projects.md:7]:
```suggestion
- [The Lost Glitches](https://playlostglitches.com/) | [Etherscan](https://etherscan.io/address/0x8460bb8eb1251a923a31486af9567e500fc2f43f) | [Twitter](https://twitter.com/TheLostGlitches)
```

File: projects.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+- Standard Protocol | Etherscan | [Twitter](https://twitter.com/standardweb3)
```
Comment (NeoDigi) [inline projects.md:8]:
```suggestion
- [Standard Protocol](https://standard.tech/) | Etherscan | [Twitter](https://twitter.com/standardweb3)
```

File: projects.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+- Kitty Crypto Gang | Etherscan | [Twitter](https://twitter.com/KittyCryptoGang)
```
Comment (NeoDigi) [inline projects.md:9]:
```suggestion
- [Kitty Crypto Gang](https://www.kittycryptogang.com/) | Etherscan | [Twitter](https://twitter.com/KittyCryptoGang)
```

File: projects.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+- X Rabbits Club | Etherscan | [Twitter](https://twitter.com/XRabbitsClub)
```
Comment (NeoDigi) [inline projects.md:10]:
```suggestion
- [X Rabbits Club](https://xrabbits.club/) | [Etherscan](https://etherscan.io/address/0x534d37c630b7e4d2a6c1e064f3a2632739e9ee04) | [Twitter](https://twitter.com/XRabbitsClub)
```

**Review by NeoDigi — COMMENTED** [review]
Added some links

**Comment by cygaar** [comment]
Thank you @NeoDigi !

**Pull request merged**